website/docs/d/fargate_workload_agent.md

```diff
@@ -32,7 +31,6 @@ data "sysdig_fargate_workload_agent" "instrumented_containers" {
 ## Argument Reference
 
 * `container_definitions` - (Required) The input Fargate container definitions to instrument with the Sysdig workload agent.
-* `sysdig_access_key` - (Required) The Sysdig Access Key (Agent token).
```

Comment:
I think we should keep this in the docs as "optional" since we do still consume it. We may have to resurrect this feature in the future.
This isn't a mandatory ask from me, happy either way.

Reply:
I prefer removing it from the doc, at least for now, as we did for the collector host/port a while ago.
There's no point in passing the access key to the workload agent ATM unless you want to use the direct connection mode.

Reply:
Fair enough, happy to remove it.